collections/openapi/schemas/collectionInfo.yaml

```diff
@@ -19,7 +19,7 @@ properties:
   extent:
     $ref: extent.yaml
   itemType:
-    description: indicator about the type of the items in the collection (the default value is 'unknown').
+    description: indicator about the type of the items in the collection if the collection has an accessible /collections/{collectionId}/items endpoint
     type: string
     default: unknown
```

Comment:
Note that "unknown" would make it impossible to add a dependency from Features to Common in a future revision of Features Part 1. See #107. Maybe we should resolve that issue in the discussion, too.